### PR TITLE
Testing test

### PR DESCRIPTION
**Pull request by wendys-cats**
testing test for @tsusanka. Please look at it!

### CODE REVIEW

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (tsusanka) [inline packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: Please put the file also under the `suite` folder instead of `dashboard` so this is inline. (I am also not sure this runs on the CI if it does not match). Also rename the file from `New` to `new` as we use lowercase for file names.

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (tsusanka) [inline packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: Let's always add a space after the `//`. So `// Initializes Trezor (random seed)`. Let's do this everywhere.

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (tsusanka) [inline packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: - Space after //
- Please replace `“` with `"`. I know it is silly but it is common to use `"` instead

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (tsusanka) [inline packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: Let's drop this comment. It is fine to do it this way.

File: packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx
Comment (tsusanka) [inline packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx]: Typo `omboarding` -> `onboarding`.

**Review by tsusanka — COMMENTED** [review]
Thanks! Some first review coming:

- I will not bike-shed about the formatting as we should enable it for this package as well and check automatically (#4054).
- Please apply the review changes in a separate fixup commit as described here https://docs.trezor.io/trezor-firmware/misc/review.html.

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (tomasklim) [reply packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: Moreover please add `.test` to the file name => `new-account-message.test.ts`

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (tomasklim) [inline packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]:
```suggestion
describe('New accounts', () => {
```
Let's use this name as name of test case

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (tomasklim) [reply packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: I think that this comment with some changes should be inside `it('xxxxx', () => {`

File: packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx
Comment (tomasklim) [reply packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx]:
```suggestion
                        data-test="@onboarding/device-acquire"
```
or 
```suggestion
                        data-test="@device-acquire"
```
not sure if the initial screen is really "onboarding"

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx
Comment (tomasklim) [inline packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx]:
```suggestion
        <StyledCard data-test="@accounts/empty-account/default-native-account">
```

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty/index.tsx
Comment (tomasklim) [inline packages/suite/src/views/wallet/transactions/components/AccountEmpty/index.tsx]:
```suggestion
                        data-test="@accounts/empty-account/receive"
```

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty/index.tsx
Comment (tomasklim) [inline packages/suite/src/views/wallet/transactions/components/AccountEmpty/index.tsx]:
```suggestion
                        data-test="@accounts/empty-account/buy"
```

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx
Comment (tomasklim) [inline packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx]:
```suggestion
                    data-test="@accounts/empty-account/default-native-account/close"
```

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx
Comment (tomasklim) [reply packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx]: @tsusanka this is confusing, that it is called Bech32Banner but there is something about default accounts.

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (tomasklim) [inline packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]:
```suggestion
        cy.getTestElement('@onboarding-device-acquire').click();
```

**Review by tomasklim — CHANGES_REQUESTED** [review]
I don't know if I should have checked it, but here are some suggestions

**Comment by mroz22** [comment]
Few notes: 
- we should try to avoid cy.wait() whenever possible. It is better to wait by looking for some element that you know should be avaialable when some condition becomes true. https://docs.cypress.io/guides/core-concepts/retry-ability
- you don't want to deal with unacquired device screen here. What you really want is to wait for discovery to finish (you can check other test files how this is done, for example passhprase related). Once discovery is finished a device is released, you should not see unacquired device page after reloading (I hope)

File: packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx
Comment (mroz22) [reply packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx]: yes. better to follow the pattern set here https://github.com/trezor/trezor-suite/blob/b6e27b1d8243ff7c9a5aab9bd1e92bd117339bcb/packages/suite/src/views/suite/device-seedless/index.tsx#L8 for all components within "PrerequisitesGuide" folder

**Comment by tsusanka** [comment]
@wendys-cats is on vacation till Wednesday/Thursday so we'll continue this then.

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (wendys-cats) [reply packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: 534889de5464e01a3eb5970394719506dc079b1e done

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (wendys-cats) [reply packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: 534889de5464e01a3eb5970394719506dc079b1e done

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (wendys-cats) [reply packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: 534889de5464e01a3eb5970394719506dc079b1e done

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (wendys-cats) [reply packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: 534889de5464e01a3eb5970394719506dc079b1e done

File: packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx
Comment (wendys-cats) [reply packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx]: 534889de5464e01a3eb5970394719506dc079b1e done

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (wendys-cats) [reply packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: 534889de5464e01a3eb5970394719506dc079b1e done

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx
Comment (wendys-cats) [reply packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx]: 534889de5464e01a3eb5970394719506dc079b1e done

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty/index.tsx
Comment (wendys-cats) [reply packages/suite/src/views/wallet/transactions/components/AccountEmpty/index.tsx]: 534889de5464e01a3eb5970394719506dc079b1e done

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty/index.tsx
Comment (wendys-cats) [reply packages/suite/src/views/wallet/transactions/components/AccountEmpty/index.tsx]: 534889de5464e01a3eb5970394719506dc079b1e done

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx
Comment (wendys-cats) [reply packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx]: 534889de5464e01a3eb5970394719506dc079b1e done

File: packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts
Comment (wendys-cats) [reply packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts]: 534889de5464e01a3eb5970394719506dc079b1e done

**Comment by tsusanka** [comment]
@wendys-cats you have accidently committed changes which you probably did not want to. See the [changes](https://github.com/trezor/trezor-suite/pull/4053/commits/534889de5464e01a3eb5970394719506dc079b1e).

**Comment by wendys-cats** [comment]
> @wendys-cats you have accidently committed changes which you probably did not want to. See the [changes](https://github.com/trezor/trezor-suite/pull/4053/commits/534889de5464e01a3eb5970394719506dc079b1e).

@tsusanka  You are right, sorry, looks like some wild fonts!

Deleted with 26b201d83ba800074a5d28db1c7b213ebddfd302

File: packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx
Comment (tsusanka) [reply packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx]: @mroz22 so what @wendys-cats did in 534889d is fine right?

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx
Comment (tsusanka) [reply packages/suite/src/views/wallet/transactions/components/AccountEmpty/components/Bech32Banner/index.tsx]: @tomasklim you are right but that's probably out of scope for this PR..

**Comment by tsusanka** [comment]
Great, looks good now. Let's wait for final review from @mroz22 or @tomasklim and I think we are good to go.

File: packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts
Comment (mroz22) [inline packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts]: Doing this on line 12

```
cy.prefixedVisit('/');
```

and then 
```
cy.prefixedVisit('/accounts/');
```

has no added value. cy.prefixedVisit does full reload. You don't want to do that. I think you want to replace line 19 with 
```
cy.getTestElement('@suite/menu/wallet-index').click();
```

File: packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts
Comment (mroz22) [inline packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts]: Any special reason for having these two same lines?

File: packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts
Comment (mroz22) [inline packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts]: unacquired device. This is something I would like to test in a separate test as this is not a happy path. Unacquired device happens because of doing full reload on line 19. Once you will resolve the first comment you will not need this line 35 at all. 

But anyway, great idea to test unacquired device by doing reload during discovery!

**Review by mroz22 — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx
Comment (mroz22) [reply packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceAcquire.tsx]: I think it is acceptable.

File: packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts
Comment (wendys-cats) [reply packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts]: cb0a3c4762e4b4db44b9de5a6c79d524fdc8e310 done

File: packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts
Comment (wendys-cats) [reply packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts]: duplicity, deleted with cb0a3c4762e4b4db44b9de5a6c79d524fdc8e310

File: packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts
Comment (wendys-cats) [reply packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts]: cb0a3c4762e4b4db44b9de5a6c79d524fdc8e310 done

**Review by mroz22 — APPROVED** [review]
looks good. :+1:

**Comment by tsusanka** [comment]
@wendys-cats please squash and force push ([docs](https://docs.trezor.io/trezor-firmware/misc/review.html) if needed).

**Comment by tsusanka** [comment]
@vdovhanych we are getting again the `ERROR: for test-run  error parsing HTTP 404 response body: unexpected end of JSON input: ""` error again and I have retried like five times :/. Any ideas?

**Comment by vdovhanych** [comment]
> @vdovhanych we are getting again the `ERROR: for test-run  error parsing HTTP 404 response body: unexpected end of JSON input: ""` error again and I have retried like five times :/. Any ideas?

The only thing we can do is to remove gitlab dependency proxy and pull from docker directly until the gitlab sorts it out.

**Comment by vdovhanych** [comment]
Here is the link to the issue https://gitlab.com/gitlab-org/gitlab/-/issues/337825

**Comment by tsusanka** [comment]
@vdovhanych I am just puzzled that 1. This does not happen on other branches 2. The job deterministically fails - I rerun it 10x times and it failed every time. So it does not seem random per-se. But maybe it is some caching weirdness on Gitlab side :shrug:. So do you think we should merge and hope for the best? It is true that I do not see how this PR could introduce such error.

**Comment by vdovhanych** [comment]
@tsusanka I would not hold up this pr because of this issue. You're right that this pr has nothing that would introduce this kind of error.

**Comment by tsusanka** [comment]
Very well then!

**Comment by tsusanka** [comment]
Damn, I wanted to squash first. Well apologies.

**Pull request merged**
